x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/components/custom_urls/index.scss

```diff
@@ -0,0 +1,12 @@
+.ml-custom-urls-selection {
+  > .euiFlexGroup {
+    > .euiFlexItem {
+      &:first-child {
+        max-width: 388px;
+      }
+      &:last-child {
+        flex-basis: 50%;
```

Comment:
I really didn't want to add custom CSS rules, but `EuiDescribedFormGroup` doesn't provide any props to modify `EuiFlexItem` width.